Controllers/auth/verifyUser.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const jwt = require('jsonwebtoken');
+
+const verifyAccessToken = (req, res, next) => {
+  if (req.headers.access_token) {
+    const pubKey = fs.readFileSync('../../rsa_secret.pub');
+    jwt.verify(req.headers.access_token, pubKey, (error, decoded) => {
+      if (error) {
+        res.status(200).json({
+          success: false,
+          error,
+          results: null,
+        });
+        return;
+      }
+      req.body.username = decoded.username;
```

Comment:
I guess if we store id, then it would be better.

Reply:
Id is like the primary key right...wouldnt it be a safety issue if we give out id..since jwt can be easily decoded..

Reply:
No there won't be any safety issue, id can be visible on profile too.. But it's ok, if you want to use username, you can.